profiles/common/modules/features/nexteuropa_newsroom/classes/NewsroomHelper.inc

```diff
@@ -0,0 +1,633 @@
+<?php
+
+/**
+ * @file
+ * Code for the NewsroomHelper class.
+ */
+
+/**
+ * NewsroomHelper class.
+ */
+class NewsroomHelper {
+
+  /**
+   * Merges a term_tid and its children into one array.
+   *
+   * @param int $term_tid
+   *   Term id.
+   * @param int $term_vid
+   *   Vid id.
+   *
+   * @return array.
+   *   tids
+   */
+  public static function getTermChildren($term_tid, $term_vid = NULL) {
+    // We need parent ID also in the output array to use it in query.
+    $parent_children_tids = array($term_tid);
+    $children_tids = array_keys(taxonomy_get_children($term_tid, $term_vid));
+    if (count($children_tids) > 0) {
+      $parent_children_tids = array_merge($parent_children_tids, $children_tids);
+    }
+
+    return $parent_children_tids;
+  }
+
+  /**
+   * Helper function to serialize PDO object into an array.
+   */
+  public static function convertPdoToArray($query) {
```

Comment:
the function is useless, just use http://php.net/manual/en/pdostatement.fetchall.php in the places where you call this function.